src/fidesops/task/graph_task.py

```diff
@@ -568,10 +569,13 @@ def run_access_request(
     graph: DatasetGraph,
     connection_configs: List[ConnectionConfig],
     identity: Dict[str, Any],
+    session: Session,
 ) -> Dict[str, List[Row]]:
     """Run the access request"""
     traversal: Traversal = Traversal(graph, identity)
-    with TaskResources(privacy_request, policy, connection_configs) as resources:
+    with TaskResources(
+        privacy_request, policy, connection_configs, session
+    ) as resources:
```

Comment:
We're passing in the Session here to TaskResources so we can use it to create the ExecutionLogs